wire-compiler/src/main/java/com/squareup/wire/WireCompiler.kt

```diff
@@ -188,6 +197,8 @@ class WireCompiler internal constructor(
     private const val JAVA_OUT_FLAG = "--java_out="
     private const val KOTLIN_OUT_FLAG = "--kotlin_out="
     private const val SWIFT_OUT_FLAG = "--swift_out="
+    private const val CUSTOM_OUT_FLAG = "--custom_out="
```

Comment:
let me know if you want me to have one parameter instead of two

Reply:
I guess we dont really need a `custom_out` param? Any cases you can think of that this can be useful?

Reply:
Not ideal but let's keep it